src/plugins/uiState.js

```diff
@@ -0,0 +1,27 @@
+let store;
+
+const set = (key, identifier, value) => {
+	store?.commit("uiState/set", {
+		key,
+		identifier,
+		object: value,
```

Comment:
vllt, statt `value` (impliziert einfache daten) und `object` (verwirrend zu Object) einfach `data` nutzen?
```suggestion
const set = (key, identifier, data) => {
	store?.commit("uiState/set", {
		key,
		identifier,
		data,
```